src/Card/Card.tsx

```diff
@@ -0,0 +1,100 @@
+/** @jsx jsx */
+import * as colors from "../colors";
+import { base } from "../typography";
+import { jsx } from "@emotion/core";
+import React from "react";
+import PropTypes from "prop-types";
+
+interface Props {
+  children?: React.ReactNode;
+  title?: React.ReactNode;
+  className?: string;
+  description?: React.ReactNode;
+  titleChildren?: React.ReactNode;
+  forceNoChildPadding?: boolean;
+  size?: "standard" | "large";
+}
+
+export const Card: React.FunctionComponent<Props> = ({
+  children,
+  forceNoChildPadding,
+  title,
+  titleChildren,
+  description,
+  size,
+  ...otherProps
+}) => (
+  <div
+    {...otherProps}
+    css={{
+      backgroundColor: colors.white,
+      color: colors.black.base,
+      boxShadow: `0 4px 8px 0 rgba(0, 0, 0, .04)`,
+      borderRadius: `0.5rem`,
```

Comment:
I don't think should be in `rem`s because border radii probably shouldn't change if the page's font size changes.